resources/js/views/Dashboard.vue

```diff
@@ -4,9 +4,11 @@
       <ControlGroup></ControlGroup>
     </div>
```

Comment:
to avoid duplicate code put `DashboardNotification` component outside of the checks for `comparison` and `intervention` and do a `v-else` in side of the div where you're checking if a user is in control group

Reply:
You can add all these into one `div`

Reply:
```
    <div>
      <DashboardNotification v-if="this.user.user_group !=='control'"></DashboardNotification>
      <ControlGroup v-if="this.user.user_group === 'control'"></ControlGroup>
      <ComparisonGroup v-if="this.user.user_group === 'comparison'"></ComparisonGroup>
      <InterventionGroup v-if="this.user.user_group === 'intervention'"></InterventionGroup>
    </div>
```

Reply:
I think you should also make a getter for this as well call it `getter.userGroup`